.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: NIDAQmx package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Don't do this. It doesn't install into the right venv and it doesn't use the versions listed in poetry.lock.

ni-python-styleguide already installs flake8.

pyproject.toml already specifies pytest.

This repo doesn't have a requirements.txt.